rust/relay/ebpf-turn-router/src/main.rs

```diff
@@ -72,22 +72,25 @@ static UDP_TO_CHAN_64: HashMap<PortAndPeerV6, ClientAndChannelV4> =
 
 #[xdp]
 pub fn handle_turn(ctx: XdpContext) -> u32 {
-    trace!(
-        &ctx,
-        "udp-checksumming = {}, allocation-range = {}..={}",
-        if config::udp_checksum_enabled() {
-            "true"
-        } else {
-            "false"
-        },
-        *config::allocation_range().start(),
-        *config::allocation_range().end(),
-    );
+    // Can cause verifier issues on some older kernls. Uncomment if you really need to print this
```

Comment:
This was causing verifier issues, and I'm not sure how useful it is on prod?

Reply:
Yeah we can remove it. I used this to test that the config gets pushed correctly.